buildSrc/src/main/java/com/uber/okbuck/core/manager/DependencyManager.java

```diff
@@ -98,6 +104,8 @@ public void resolveCurrentRawDeps() {
       return;
     }
 
+    dependencyExporter.export(rawDependencies);
```

Comment:
we can move this to the start of `finalizeDependencies` which is responsible to generate all the output.

Reply:
moved to finalizeDependencies